client/src/components/controls/Controls.vue

```diff
@@ -116,35 +111,35 @@ export default {
               v-else
               icon
               small
-              title="Space key"
+              title="(space) Pause"
               @click="annotator.$emit('pause')"
             >
               <v-icon>mdi-pause</v-icon>
             </v-btn>
             <v-btn
               icon
               small
-              title="Right key"
+              title="(f, right-arrow) next frame"
               @click="nextFrame"
             >
               <v-icon>mdi-skip-next</v-icon>
             </v-btn>
           </v-col>
           <v-col
-            cols="4"
-            md="4"
-            sm="2"
-            xs="0"
-            class="py-1"
-          />
+            class="pl-1 py-1"
+            align="right"
+          >
+            <v-btn
+              icon
+              small
+              title="(r)eset pan and zoom"
+              @click="annotator.$emit('reset-zoom')"
+            >
+              <v-icon>mdi-image-filter-center-focus</v-icon>
+            </v-btn>
+          </v-col>
         </v-row>
       </v-container>
     </v-toolbar>
   </div>
 </template>
-
-<style lang="scss" scoped>
-.controls-row {
-  min-width:420px;
-}
-</style>
```

Comment:
This is really a minor thing and can be ignored.
Removing this and not replacing it with something similar, whether that be a min-width for the v-toolbar or another item causes this at small widths:
![image](https://user-images.githubusercontent.com/61746913/91082721-0faf3b80-e617-11ea-9ac5-da8297eae6cf.png)
If you add in a width it will behave more like the toolbar at the top of the app where items are pushed offscreen instead of wrappingr.  There might be some no-wrapping arguments that can be used to do this as well:
![image](https://user-images.githubusercontent.com/61746913/91082799-32d9eb00-e617-11ea-8b6b-e261feb345ce.png)